cmake_min_version.py

```diff
@@ -155,16 +218,18 @@ def full_search(cmake_parameters: List[str], tools_dir: str) -> Optional[CMakeBi
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Find the minimal required CMake version for a project.')
     parser.add_argument('params', type=str, nargs='+', help='parameters to pass to CMake')
+    parser.add_argument('--latest_patch', action='store_true',
```

Comment:
I first was confused why this parameter is here and not in `cmake_downloader.py`. Maybe the help string should clarify that the files are downloaded in case the version was not found in the tools directory?